app/js/lib/server/index.js

```diff
@@ -21,7 +21,7 @@ const p = Object.freeze({
 
 export default class Server extends EventDispatcher {
   constructor({ settings, net } = {}) {
-    super(['online', 'push-message']);
+    super(['logged-in', 'online', 'push-message']);
```

Comment:
that was actually my first try but I thought "login" was ambiguous. Is it when starting the login request ? Is it when ending it ? That's why I choose "logged-in". If you don't like it, how about "login-success" ?